Sources/_StringProcessing/Unicode/Decoding.swift

```diff
@@ -33,13 +33,13 @@
 
 enum UnsafeAssumingValidUTF8 {
   @inlinable @inline(__always)
-  public func decode(_ x: UInt8) -> Unicode.Scalar {
+  public static func decode(_ x: UInt8) -> Unicode.Scalar {
```

Comment:
Part of moving these into the `_StringProcessing` module should be dropping all `public` keywords, since `_StringProcessing` will be supplying API as well.